lib/locales/en.ts

```diff
@@ -926,10 +959,12 @@ export const locale: Locale = {
     sebastian_de_aparicio_religious: 'Blessed Sebastian de Aparicio, Religious',
     sebastian_of_milan_martyr: 'Saint Sebastian, Martyr',
     senan_of_inis_cathaigh_bishop: 'Saint Senan, Bishop',
+    senerius_bishop: 'Saint Senerius, Bishop',
```

Comment:
In this case, I might be more inclining to use `avranches` instead of `normandy`, as we already use that city name in other `.names` keys and even the French celebration name uses it. :thinking:

However, choose which you prefer, you can keep it as it is. YMMV.